File: Backend/BikeRentalCore/BikeRentalCore/DTOs/In/RentRequest.cs
using BikeRentalCore.Enums;

namespace BikeRentalCore.DTOs.In;

public record RentRequest(Guid BikeId, 
                          Guid PointId, 
                          PaymentMethod PaymentMethod, 
                          string PaymentData,
                          int RentedDays);

File: BikeRentalsMobileApp/app/src/main/java/com/rtlocation/bike/models/RentRequest.kt
package com.rtlocation.bike.models

import kotlinx.serialization.Serializable

@Serializable
data class RentRequest(val bikeId: String,
                       val pointId: String,
                       val paymentMethod: Int,
                       val paymentData: String,  // JWE
                       val rentedDays: Int)

